DOCS about DHIS2 Live is outdated (#1065)
dhis2/dhis2-docs: src/user/about-demo-server-live-package-and-database-design.md

> important work on this server. It is strictly for demonstration
> purposes on only\!

## Spinning up a local DHIS2 instance using D2 Cluster  { #mod2_1 } 

### Running D2 Cluster

> Using Docker containers to run a d2 culster is the easiest way to get started with DHIS2 on

Reviewer: ```suggestion
Using Docker containers to run a d2 cluster is an easy way to get started with DHIS2 on
```

your local computer. This approach is appropriate for a stand-alone
installation and demos. Simply follow the guide
[here](https://developers.dhis2.org/docs/guides/spin-up-local-instance).

#### Prerequisites to running D2 Cluster

You must be sure that you have installed [Yarn](https://classic.yarnpkg.com/en/docs/install/) and [Nodejs](https://nodejs.org/en/). A current version of [Docker](https://www.docker.com/) must be

> installed on your machine as well. Please refer to the guide for detailed information [guide](https://developers.dhis2.org/docs/guides/spin-up-local-instance).

Reviewer: ```suggestion
installed on your machine as well. Please refer to the [guide](https://developers.dhis2.org/docs/guides/spin-up-local-instance) for detailed information.
```
